setup.cfg

```diff
@@ -55,6 +55,7 @@ image =
 jpeg2000 =
   glymur>=0.8.18,!=0.9.0,!=0.9.5
 map =
+  contourpy>=1
```

Comment:
This should now be `contourpy>=1.0.1`. I had an unnecessary `numpy` version pin in `contourpy` which might cause problems, the new release fixes this.